ariadne/utils.py

```diff
@@ -0,0 +1,222 @@
+import torch
+import numpy as np
+import itertools
+import pandas as pd
+from copy import deepcopy
+from ariadne.tracknet_v2.metrics import point_in_ellipse
+import faiss
+import matplotlib.pyplot as plt
+from scipy.interpolate import make_interp_spline, BSpline
+import os
+
+def fix_random_seed(seed):
```

Comment:
This code is deprecated! I removed `fix_random_seed` centuries ago. Use this:
`from pytorch_lightning import seed_everything`

Reply:
btw, i don't see any usage of this function